instrumentation/opentelemetry-instrumentation-asgi/MANIFEST.in

```diff
@@ -0,0 +1,9 @@
+graft src
+graft tests
+global-exclude *.pyc
+global-exclude *.pyo
+global-exclude __pycache__/*
+include CHANGELOG.md
```

Comment:
Do we need this line? I don't see it in this package. Same for all the other `MANIFEST.in` files?